### PR TITLE
Install IonCube Loader on CentOS 6

### PR DESCRIPTION
**Pull request by ALBDX**

### CODE REVIEW

**Comment by ALBDX** [comment]
Sorry If I missed something I haven't write a tutorial on GitHub before

**Comment by n-se** [comment]
Thank you very much for your tutorial. I fixed some typos and changed the single line codeblocks to a single ` in front of the command and behind of it.

Please add the headings from the [template file](https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md) and also the license block at the bottom of the template to your tutorial.
You can also remove your title in the tutorial as it will be displayed on the community page through the headings.

**Comment by ALBDX** [comment]
I have added the headings and the license block

File: tutorials/how-to-install-ioncube-loader-centos-6 [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+SPDX-License-Identifier: MIT
+path: "/tutorials/how-to-install-ioncube-loader-centos-6"
+slug: "how-to-install-ioncube-loader-centos-6"
+date: "2019-03-24"
+title: "How to install IonCube Loader on CentOS 6"
+short_description: "This is a tutorial will explain How to install IonCube Loader on CentOS 6"
+tags: ["CentOS", "Linux", "IonCube Loader","PHP Encoder"]
+author: "Albin Pllana"
+author_link: "https://github.com/ALBDX"
+author_img: "https://avatars3.githubusercontent.com/u/31274970?s=460&v=4"
```
Comment (n-se) [inline tutorials/how-to-install-ioncube-loader-centos-6]:
```suggestion
author_img: "https://avatars3.githubusercontent.com/u/31274970"
```

File: tutorials/how-to-install-ioncube-loader-centos-6 [hunk 2]
```diff
@@ -0,0 +1,122 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/how-to-install-ioncube-loader-centos-6"
+slug: "how-to-install-ioncube-loader-centos-6"
+date: "2019-03-24"
+title: "How to install IonCube Loader on CentOS 6"
+short_description: "This is a tutorial will explain How to install IonCube Loader on CentOS 6"
```
Comment (n-se) [inline tutorials/how-to-install-ioncube-loader-centos-6]:
```suggestion
short_description: "This tutorial will explain how to install the IonCube Loader on CentOS 6"
```

**Comment by ALBDX** [comment]
Thanks for help

**Comment by n-se** [comment]
You can commit these two suggestions from the review yourself. Just click "Commit suggestions" to apply them. After that I think it is ready!

**Comment by ALBDX** [comment]
Done

**Comment by Billito00** [comment]
Please review the introduction again. The last sentence makes no sense.  
Also please write in present tense instead of future tense.

**Comment by ALBDX** [comment]
It is okay now?

**Comment by Billito00** [comment]
It's fine. Thank you.

**Pull request merged**